pyms/flask/app/create_app.py

```diff
@@ -192,11 +201,32 @@ def init_metrics(self) -> None:
         if self.metrics:
             self.application.register_blueprint(self.metrics.metrics_blueprint)
             self.metrics.add_logger_handler(
-                self.application.logger,
-                self.application.config["APP_NAME"]
+                self.application.logger, self.application.config["APP_NAME"]
             )
             self.metrics.monitor(self.application.config["APP_NAME"], self.application)
 
+    def init_opentelemetry(self) -> None:
+        if self.opentelemetry:
+            if self.opentelemetry.config.metrics.enabled:
+                # Set metrics backend
+                self.opentelemetry.set_metrics_backend()
+                # Set the metrics blueprint
+                # DISCLAIMER this endpoint may be only necessary with prometheus client
+                self.application.register_blueprint(self.opentelemetry.blueprint)
+                # Set instrumentations
+                if self.opentelemetry.config.metrics.instrumentations.flask:
```

Comment:
@avara1986 I'm having some doubts defining the opentelemetry service. 
For example, in this section. There may be tens of instrumentations: aiohttp, flask, sqlite3, jinja, mysql, ... There's already more than [15](https://opentelemetry-python.readthedocs.io/en/stable/instrumentation/instrumentation.html). This means that in this section, there will grow a lot. 
My question is, should this conditionals be directly on the service? Some of them have to be here, but only a couple or so. This would make this section lighter in the future, but this seems to not be the way the rest of the services are doing this, so maybe for consistency this shouldn't.